File: relatorio/codigos/ruby/micro04.rb
# Le numero e informa quais estao entre 10 e 150

intervalo = 0
for i in 1..5
  puts("Digite um numero:")
  num = gets.chomp.to_i
  if num >= 10
    if num <= 150
      intervalo = intervalo + 1
    end
  end
end

puts("Ao total, foram digitados #{intervalo} numeros no intervalo entre 10 e 150")

File: relatorio/codigos/lua/micro04.lua
-- Listagem 16: Le numeros e informa quais estao entre 10 e 150

--[[ Função: Ler 5 numeros e ao final informar quantos numeros estao no intervalo entre 10 (inclusive) e 150(inclusive) --]]

intervalo = 0

for x=1,5,1
do
	print("Digite um número")
	num = io.read("*number")
	if(num >= 10)
	then
		if(num <= 150)
		then
			intervalo = intervalo + 1
		end
	end
end

print("Ao total, foram digitados",intervalo,"números no intervalo entre 10 e 150")